packages/server/modules/core/services/objects.js

```diff
@@ -3,21 +3,6 @@ const knex = require(`@/db/knex`)
 const Objects = () => knex('objects')
 
 module.exports = {
-  async getObjects(streamId, objectIds) {
```

Comment:
only used in 1 test, where I replaced the usage with the equivalent getStreamObjects repo fn